Gruntfile.js

```diff
@@ -0,0 +1,150 @@
+      'vancouver'
+    ],
+    clean:{
+      dist:{
+        src:["**/dist/*.zip"]
+      },
+      events:{
+        src:["**/event.*.json"]
+      }
+    }
+  });
+
+  grunt.registerTask('createEvent', function(style){
+    var data = {
+      "path":"/?responseformat=json&style="+style,
+      "body":{
+        "items":[{"id":"ITEM-1","type":"Resource","title":"2","container-title":"Compilers: Principles, Techniques, and Tools (2nd Edition): Alfred V. Aho, Monica S. Lam, Ravi Sethi, Jeffrey D. Ullman: 9780321486813: Amazon.com: Books","publisher":"Addison Wesley; 2nd edition (September 10, 2006)"},{"id":"ITEM-2","type":"book","title":"The Design and Implementation of the 4.3 BSD UNIX Operating System: Samuel J. Leffler, Marshall Kirk McKusick, Michael J. Karels, John S. Quarterman, Samuel Leffler: 9780201061963: Amazon.com: Books","publisher":"Addison-Wesley; 1st edition (October 1, 1989)","URL":"http:\/\/www.amazon.com\/Design-Implementation-UNIX-Operating-System\/dp\/0201061961"},{"id":"ITEM-3","type":"Resource","title":"2","container-title":"Compilers: Principles, Techniques, and Tools (2nd Edition): Alfred V. Aho, Monica S. Lam, Ravi Sethi, Jeffrey D. Ullman: 9780321486813: Amazon.com: Books","publisher":"Addison Wesley; 2nd edition (September 10, 2006)"},{"id":"ITEM-4","type":"book","title":"The Design and Implementation of the 4.3 BSD UNIX Operating System: Samuel J. Leffler, Marshall Kirk McKusick, Michael J. Karels, John S. Quarterman, Samuel Leffler: 9780201061963: Amazon.com: Books","publisher":"Addison-Wesley; 1st edition (October 1, 1989)","URL":"http:\/\/www.amazon.com\/Design-Implementation-UNIX-Operating-System\/dp\/0201061961"},{"id":"ITEM-5","type":"book","title":"The Design and Implementation of the 4.3 BSD UNIX Operating System: Samuel J. Leffler, Marshall Kirk McKusick, Michael J. Karels, John S. Quarterman, Samuel Leffler: 9780201061963: Amazon.com: Books","publisher":"Addison-Wesley; 1st edition (October 1, 1989)","URL":"http:\/\/www.amazon.com\/Design-Implementation-UNIX-Operating-System\/dp\/0201061961"},{"id":"ITEM-6","type":"Resource","title":"2","container-title":"Compilers: Principles, Techniques, and Tools (2nd Edition): Alfred V. Aho, Monica S. Lam, Ravi Sethi, Jeffrey D. Ullman: 9780321486813: Amazon.com: Books","publisher":"Addison Wesley; 2nd edition (September 10, 2006)"}]
+      },
+      "query":{
+        "responseformat":"json",
+        "style":style
+      }
+    };
+    grunt.file.write('./event.'+style+'.json', JSON.stringify(data));
+  });
+
+  grunt.registerMultiTask('testStyles', function(){
+    var style = this.data;
+
+    console.log('Testing style:'+style);
+
+    grunt.task.run('createEvent:'+style);
+    grunt.task.run('lambda_invoke:'+style);
+  });
+
+  grunt.registerTask('test', function(){
+    grunt.task.run(['testStyles', 'clean:events']);
+  });
+  grunt.registerTask('dist', ['clean:dist', 'clean:events', 'lambda_package', 'lambda_deploy']);
```

Comment:
i think its worth separating `dist` which should really be about creating a `distribution` from `deploy` which should be about deploying a distribution.